CONTRIBUTING.md

```diff
@@ -182,6 +182,12 @@ Classes
 
 ![](docs/source/uml/classes_customer_choice.png)
 
+## 2025 CLV Roadmap
+
+![](docs/source/_static/clv_roadmap_2025.png)
+
+For [issues](https://github.com/pymc-labs/pymc-marketing/issues) to work on, search `is:issue state:open label:CLV`
```

Comment:
Just share the URL of the search of `is:issue state:open label:CLV` directly

Reply:
That URL is : https://github.com/pymc-labs/pymc-marketing/issues?q=is%3Aissue%20state%3Aopen%20label%3ACLV